src/main/tables/92_quick_launch_defaults.sql

```diff
@@ -0,0 +1,16 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE quick_launch_defaults (
+  -- Primary key
+  id uuid NOT NULL,
+
+  -- Foreign key into the quick_launches table.
+  quick_launch_id uuid NOT NULL,
+
+  -- Foreign key into the users table.
+  user_id uuid NOT NULL,
+
+  -- Whether or not it's a global default (AKA default default). Should only
+  -- be toggleable by the app creator.
+  global boolean NOT NULL DEFAULT false
+)
```

Comment:
The one thing that concerns me about this table is that I'm not sure what should happen if there's more than one default for a single app and user (or if there's more than one global default) for a single app. I'd be a little tempted to alter the database structure to prevent that from happening, which would probably require us to add an app ID column to the `quick_launch_defaults` table. With the addition of the column, we could add a uniqueness constraint on the user ID, app ID and global flag, which would prevent this scenario from happening.

Reply:
Hmm, not sure how to fix this quite yet. I'd like to avoid the duplicate app_id columns in the this and the quick_launches table, if possible.

Reply:
After consulting with Sarah, I've merged the quick_launch_defaults table into the quick_launches table. This resulted in `quick_launches.is_creator_default` and `quick_launches.is_global_default` being added. 

I renamed the `quick_launches.public` to `quick_launches.is_public` to remain consistent with the other two boolean columns that were added and to avoid confusion with the `public` schema.

I added a uniqueness constraint encompassing the `app_id`, `creator`, and `is_creator_default` columns. I added another uniqueness constraint encompassing the `app_id' and `is_global_default` columns.